clientconn.go

```diff
@@ -104,6 +104,15 @@ const (
 // DialOption configures how we set up the connection.
 type DialOption func(*dialOptions)
 
+// UseCompressor returns a CallOption which sets the compressor used when sending the request.
+// If WithCompressor is set, UseCompressor has higher priority.
+func UseCompressor(name string) CallOption {
```

Comment:
Please add:

```go
//
// This API is EXPERIMENTAL.
```

Here and to the other functions you created.  We may want to make changes if it doesn't work out right for users.

Reply:
Done. Only add comment for this function. The other two are "This package is experimental" for encoding.go and gzip.go.